### PR TITLE
lets use sha256sum instead of shasum -a 256

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

I tested it, you can see the results from the link below:
> https://github.com/developer-guy/cosign-installer/actions/runs/1142685296

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ runs:
         bootstrap_version='v1.0.0'
         expected_bootstrap_version_digest='e36a05ab402bfee5463ad4752d8dc2941204c7b01a9a9931f921e91d94ba2484'
         curl -L https://storage.googleapis.com/cosign-releases/v1.0.0/cosign-linux-amd64 -o cosign
-        shaBootstrap=$(shasum -a 256 cosign | cut -d' ' -f1);
+        shaBootstrap=$(sha256sum cosign | cut -d' ' -f1);
```
Comment (hectorj2f) [inline action.yml:21]: Why don't you replace it in https://github.com/sigstore/cosign-installer/blob/main/action.yml#L42 too ?

File: action.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ runs:
         bootstrap_version='v1.0.0'
         expected_bootstrap_version_digest='e36a05ab402bfee5463ad4752d8dc2941204c7b01a9a9931f921e91d94ba2484'
         curl -L https://storage.googleapis.com/cosign-releases/v1.0.0/cosign-linux-amd64 -o cosign
```
Comment (hectorj2f) [inline action.yml:20]: unrelated, but we might want to use here `https://storage.googleapis.com/cosign-releases/${bootstrap_version}/cosign-linux-amd64` instead. wdyt ?

**Review by hectorj2f — CHANGES_REQUESTED** [review]

File: action.yml (hunk 2 above)
Comment (developer-guy) [reply action.yml:20]: nice catch 👏

File: action.yml (hunk 1 above)
Comment (developer-guy) [reply action.yml:21]: oh ok, thank you, I missed that one.

File: action.yml [hunk 3]
```diff
@@ -48,7 +48,7 @@ runs:
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
```
Comment (hectorj2f) [inline action.yml:49]: This line is a bit scary :S. What about checking if that dependency exists before running the `update` ?

Comment (hectorj2f) [reply action.yml:49]: We could tackle this in a different PR, it is unrelated to your changes here. Just sharing my thoughts here :).

**Review by hectorj2f — COMMENTED** [review]
LGTM, perhaps we could check the existence of this dependency before running `apt-get update` https://github.com/sigstore/cosign-installer/pull/15/files#r691173829. wdyt ?

**Review by hectorj2f — APPROVED** [review]

**Review by cpanato — APPROVED** [review]
thanks for the improvements

**Pull request merged**